crates/cargo-contract/src/cmd/extrinsics/upload.rs

```diff
@@ -147,17 +147,22 @@ impl UploadCommand {
             .contracts()
             .upload_code(code, self.extrinsic_opts.storage_deposit_limit);
 
-        let result = submit_extrinsic(client, &call, signer).await?;
-
-        let call_result =
-            CallResult::from_events(&result, transcoder, &client.metadata())?;
-
-        let display = call_result.display_events(self.extrinsic_opts.verbosity()?);
-        println!("{}", display);
-
-        let code_stored = result.find_first::<api::contracts::events::CodeStored>()?;
-
-        Ok(code_stored)
+        let result = submit_extrinsic(client, &call, signer).await;
+        match result {
+            Ok(result) => {
+                let events =
+                    DisplayEvents::from_events(&result, transcoder, &client.metadata())?;
+                println!(
+                    "{}",
+                    events.display_events(self.extrinsic_opts.verbosity()?)
+                );
+                let code_stored =
+                    result.find_first::<api::contracts::events::CodeStored>()?;
+                Ok(code_stored)
+            }
+            // TODO: add json output here too
```

Comment:
Please resolve this TODO

Reply:
Not in the scope of this PR

Reply:
Even though the `upload` command is not explicitly mentioned, we should assume that the intent is to enable json output for all extrinsic commands and not just 2 out of the 3.

Reply:
Thank you :pray: